--- applet.js.orig	2026-07-27 14:18:31.518187875 +0200
+++ applet.js	2026-07-27 14:18:58.995049597 +0200
@@ -521,8 +521,8 @@
             // Original logic
             if (this.numberOfMonitors === 1) {
                 monitorWatchList = [Main.layoutManager.primaryIndex];
-            } else if (instances.length > 1 && !onPrimary) {
-                monitorWatchList = [this.panel.monitorIndex];
+            // } else if (instances.length > 1 && !onPrimary) {
+            //     monitorWatchList = [this.panel.monitorIndex];
             } else {
                /* This is an instance on the primary monitor - it will be
                 * responsible for any monitors not covered individually.  First
@@ -536,11 +536,11 @@
                     }
                     instances[i] = instances[i].panel.monitorIndex;
                 }
-                for (let i = 0; i < this.numberOfMonitors; i++) {
-                    if (instances.indexOf(i) === -1) {
-                        monitorWatchList.push(i);
-                    }
-                }
+                // for (let i = 0; i < this.numberOfMonitors; i++) {
+                //     if (instances.indexOf(i) === -1) {
+                //         monitorWatchList.push(i);
+                //     }
+                // }
             }
         }
         this.state.set({ monitorWatchList });
