--- /usr/share/cinnamon/applets/grouped-window-list@cinnamon.org/applet.js.orig	2025-02-24 16:24:47.000000000 +0100
+++ /usr/share/cinnamon/applets/grouped-window-list@cinnamon.org/applet.js	2025-06-16 09:02:12.727351771 +0200
@@ -521,8 +521,8 @@
             // Original logic
             if (this.numberOfMonitors === 1) {
                 monitorWatchList = [Main.layoutManager.primaryIndex];
-            } else if (instances.length > 1 && !onPrimary) {
-                monitorWatchList = [this.panel.monitorIndex];
+            // } else if (instances.length > 1 && !onPrimary) {
+            //     monitorWatchList = [this.panel.monitorIndex];
             } else {
                /* This is an instance on the primary monitor - it will be
                 * responsible for any monitors not covered individually.  First
@@ -537,9 +537,9 @@
                     instances[i] = instances[i].panel.monitorIndex;
                 }
                 for (let i = 0; i < this.numberOfMonitors; i++) {
-                    if (instances.indexOf(i) === -1) {
-                        monitorWatchList.push(i);
-                    }
+                    // if (instances.indexOf(i) === -1) {
+                    monitorWatchList.push(i);
+                    // }
                 }
             }
         }
